tests/test_style.rs

```diff
@@ -182,6 +182,55 @@ fn test_stylesheet() {
   );
 }
 
+#[test]
+fn test_stylesheet_builder() {
```

Comment:
Replace w/ 
`fn test_stylesheet_builder() -> CommonResult<()> {`
since we don't panic anymore & are using error handling.